lib/src/request/request.rs

```diff
@@ -446,7 +446,7 @@ impl<'r> Request<'r> {
     /// ```
     pub fn format(&self) -> Option<&MediaType> {
         static ANY: MediaType = MediaType::Any;
-        if self.method.supports_payload() {
+        if self.method.get().supports_payload() {
```

Comment:
Don't use `self.method.get()`. Instead, use `self.method()`. Make this change everywhere else `self.method.get()` is used.